packages/hub/services/email-card-drop-router.ts

```diff
@@ -0,0 +1,31 @@
+import Router from '@koa/router';
+import Koa from 'koa';
+import { route } from '@cardstack/hub/routes';
+
+export default class EmailCardDropRouter {
```

Comment:
Why do we need this as a router on its own?

Reply:
It’s not under `/api` 😞